src/bitcoin-tx.cpp

```diff
@@ -31,6 +31,41 @@ static bool fCreateBlank;
 static std::map<std::string,UniValue> registers;
 static const int CONTINUE_EXECUTION=-1;
 
+void SetupArgs()
```

Comment:
Nit: More suitable name could be `static void SetupTxArgs`, including the `static` after #13163